test/testlib/nuodb_database_utilities.go

```diff
@@ -747,3 +758,18 @@ func SuspendDatabaseBackupJobs(t *testing.T, namespaceName string, domain, dbNam
 		}
 	}
 }
+
+// Create a snapshot of a pvc and wait for it to become ready
+func SnapshotVolume(t *testing.T, namespaceName string, pvcName string, snapName string) {
+	kubectlOptions := k8s.NewKubectlOptions("", "", namespaceName)
+	k8s.KubectlApplyFromString(t, kubectlOptions, fmt.Sprintf(SNAPSHOT_TEMPLATE, snapName, VOLUME_SNAPSHOT_CLASS, pvcName))
+
+	AddTeardown(TEARDOWN_SNAPSHOT, func() {
+		k8s.RunKubectl(t, kubectlOptions, "delete", "volumesnapshot", snapName)
+	})
+
+	Await(t, func() bool {
+		out, error := k8s.RunKubectlAndGetOutputE(t, kubectlOptions, "get", "volumesnapshot", snapName, "-o", "custom-columns=:.status.readyToUse", "--no-headers")
```

Comment:
Why don't you use `-o jsonpath='{.status.readyToStatus}'`?

Reply:
I did not use it because kubectl needs better docs.